web-ts/src/constants/buildConfig.ts

```diff
@@ -1,5 +1,7 @@
 import type { WebBuildConfigType } from 'build-configs/BuildConfigType'
 
-const buildConfig = (): WebBuildConfigType => __BUILD_CONFIG__
+const buildConfig = (): WebBuildConfigType => {
+  return __BUILD_CONFIG__
+}
```

Comment:
This somehow fixes a syntax error and I have absolutely no clue why...

Reply:
what syntax error? Cannot reproduce ^^